app/controllers/AlertController.java

```diff
@@ -61,6 +72,38 @@ public AlertController(final Configuration configuration, final AlertRepository
     }
 
     /**
+     * Adds an alert in the alert repository.
+     *
+     * @return Created if the alert was created successfully, an error code otherwise.
```

Comment:
This comment does not match behavior.

https://en.wikipedia.org/wiki/List_of_HTTP_status_codes

Please check all documentation given the changes these endpoints have gone through.